sway-core/src/semantic_analysis/ast_node/expression/typed_expression_variant.rs

```diff
@@ -16,6 +16,7 @@ pub(crate) struct ContractCallMetadata {
 #[derivative(Eq)]
 pub(crate) enum TypedExpressionVariant {
     Literal(Literal),
+    /// a zero-sized type-system-only compile-time thing that is used for constructing ABI casts.
```

Comment:
Is this comment meant to be added before `AbiName(AbiName)`?

Reply:
yep, thanks for catching that!